desk/lib/dm.hoon

```diff
@@ -93,6 +93,11 @@
   ^-  (unit (unit cage))
   =*  on   on:writs:c
   ?+    pole  [~ ~]
+  ::
+      [%exists author=@ time=@ ~]
```

Comment:
This makes sense. The issue here is the first part of these scry paths are listed in the `%chat` `+peek` arm, which all have prepended `%x` cares. I think it would be awkward if we had something like `/x/dm/writs/u/writ/id/ship/time`. I could change the paths going to `dm.hoon` to delay the care (e.g. `/dm/writs/<care>/<id>/<ship/<time>`), but would adjusting this cause cascading breaks?